packages/amazonq/test/e2e/amazonq/VET.test.ts

```diff
@@ -2,61 +2,129 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
+import { Workbench, By, WebviewView, WebElement } from 'vscode-extension-tester'
+import { until } from 'selenium-webdriver'
 
-import { Workbench, By, EditorView, WebviewView, WebElement } from 'vscode-extension-tester'
-
-describe('Amazon Q Login Flow', function () {
+describe('Amazon Q E2E UI Test', function () {
     // need this timeout because Amazon Q takes awhile to load
-    this.timeout(30000)
-    let webviewView: WebviewView
 
-    // NOTE: I tested all the timeouts and they are necessary for the webview to load properly
+    // need this timeout
+    this.timeout(150000)
+    let webviewView: WebviewView
+    let workbench: Workbench
     before(async function () {
-        const workbench = new Workbench()
+        workbench = new Workbench()
         await workbench.executeCommand('Amazon Q: Open Chat')
 
-        await new Promise((resolve) => setTimeout(resolve, 15000))
+        // need this timeout
```

Comment:
I know its somewhat of a mystery, but it could be worth investigating why we need these timeouts? Maybe we can find another example of someone running into the same issue and link that here.